### ISSUE TITLE
Packaging script adds .gitignore into drupal8 checkout

### ISSUE THREAD

**Issue by mradcliffe** [drud/quicksprint#4]
## Feature request:

Consider copying a .gitignore into the drupal git repo root so that the special directories are ignored and we don't accidentally generate bad patches.

This also affects `git status` commands though that is mitigated by `git status -uno`.

## What you expect to happen:

Shouldn't see vendor, settings.php, services.yml, .c9/, .idea/, .vscode/, etc... when 

The packaging script could do `cp ${REPO_DIR}/example.gitignore ${STAGING_DIR}/drupal8/.gitignore`.

**Comment by rickmanelius** [comment]
Hi @mradcliffe. It looks like example.gitignore doesn't include everything you're mentioning. Perhaps we can `cat  ./idea >>` and others as well. Overall a 👍

**Comment by mradcliffe** [comment]
Here's what I've been using so far:

```
--- example.gitignore	2018-03-10 14:12:52.000000000 -0500
+++ example.gitignore	2018-03-09 20:17:36.000000000 -0500
@@ -11,6 +11,23 @@
 # including Drupal core.
 # core
 
+# IDEs
+.idea/
+.c9/
+nbproject/
+.vs/
+.vscode/
+.project
+.metadata
+.settings/
+*.swp
+*.bak
+*.tmproj
+*.tmproject
+tmtags
+*.sublime-workspace
+*.sublime-project
+
 # Core's dependencies are managed with Composer.
 vendor
 ```

**Comment by mradcliffe** [comment]
Fixed in #18 and #9
